easybuild/easyconfigs/l/Leptonica/Leptonica-1.77.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Leptonica'
+version = '1.77.0'
+
+homepage = 'http://www.leptonica.org'
+description = """Leptonica is a pedagogically-oriented open source site containing software
+ that is broadly useful for image processing and image analysis applications."""
```

Comment:
This description talks about the site, not what the Leptonica code actually does.
Please change to something relevant for the software itself.

Reply:
good point, fixed